workspaces/liferay-sample-workspace/client-extensions/liferay-sample-etc-node/app.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+import bodyParser from 'body-parser';
+import config from './util/configTreePath.js';
+import express from 'express';
+import fetch from 'node-fetch';
+import {
+	corsWithReady,
+	liferayJWT,
+} from './util/liferay-oauth2-resource-server.js';
+import log from './util/log.js';
+
+log.info(`config: ${JSON.stringify(config, null, '\t')}`);
+
+const app = express();
+
+app.use(bodyParser.json());
+app.use(corsWithReady);
+app.use(liferayJWT);
+
+app.get(config.readyPath, (req, res) => {
+	res.send('READY');
+});
+
+app.get('/comic', async (req, res) => {
+	log.info('User %s is authorized', req.jwt.username);
+	log.info('User scopes: ' + req.jwt.scope);
+
+	const comicResponse = await fetch('https://xkcd.com/info.0.json');
```

Comment:
Would it make more sense to create a "dummy" endpoint, rather than pointing at a domain owned by someone else? Or even just request a json file with `fs.readFile` as that is also async

Reply:
Its a good point you make, but in the springboot one, we are making a request to a web api.  I agree its really silly, but I'm not sure of how to show the "real" nature of a remote request without calling a real network endpoint.  Do you have some type of "dummy data" that we could put in the sample and we could show?

Reply:
Any suggestions?

Reply:
nah it should be fine if we are explicitly trying to showcase a request to a web api